package.json

```diff
@@ -33,11 +33,14 @@
     "grunt-karma": "4.0.0",
     "grunt-stylelint": "0.16.0",
     "grunt-svgmin": "5.0.0",
+    "jest": "26.6.3",
```

Comment:
You will need to setup eslint too to run Jest. The goal of this patch is to replace the previous assertion with a test in Jest